yarn.lock

```diff
@@ -18818,6 +18818,15 @@ string-width@^3.0.0, string-width@^3.1.0:
     is-fullwidth-code-point "^2.0.0"
     strip-ansi "^5.1.0"
 
+string-width@^4.1.0, string-width@^4.2.0, string-width@^4.2.3:
```

Comment:
does the yarn.lock intentional to change? may try remove node_modules and yarn from expo/expo root again.

Reply:
Yep, I removed node modules and ran yarn again